packages/openneuro-cli/src/actions.js

```diff
@@ -72,6 +73,19 @@ const uploadDataset = (dir, datasetId, validatorOptions) => {
   }
 }
 
+const notifyUploadComplete = datasetId => {
+  console.log(
+    '=====================================================================',
+  )
+  console.log('Upload Complete')
+  console.log(
+    `To publish your dataset go to https://openneuro.org/datasets/${datasetId}`,
```

Comment:
The URL is configured, this should reference the configuration instead of being hardcoded.